src/app/regular-giving/regular-giving.component.ts

```diff
@@ -456,4 +465,12 @@ export class RegularGivingComponent implements OnInit, AfterViewInit {
 
     return !!this.paymentInfoErrorMessage;
   }
+
+  getBeneficiaryIcon(beneficiary: string) {
+    return CampaignGroupsService.getBeneficiaryIcon(beneficiary);
+  }
+
+  getCategoryIcon(category: string) {
+    return CampaignGroupsService.getCategoryIcon(category);
+  }
```

Comment:
these can be simplified a little:

```suggestion

  protected getBeneficiaryIcon = CampaignGroupsService.getBeneficiaryIcon;

  protected getCategoryIcon = CampaignGroupsService.getCategoryIcon;
```